### PR TITLE
SG-22675: Shotgrid rebranding

### PR DESCRIPTION
**Pull request by jfboismenu**
Integrates tk-core v0.20.1 and rebrands.

### CODE REVIEW

File: python/shotgun_desktop/upgrade_startup.py [hunk 1]
```diff
@@ -127,12 +127,12 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
...
-            "There is a new update of the Shotgun Desktop, but it couldn't be installed. Shotgun "
+            "ShotGrid Desktop update failed",
+            "There is a new update of the ShotGrid Desktop, but it couldn't be installed. ShotGrid "
             "Desktop will be launched with the currently installed version of the code.\n"
-            "If this problem persists, please contact Shotgun support at "
-            "support@shotgunsoftware.com.\n"
-            "Error: %s" % str(e),
+            "If this problem persists, please <a href='%s'>contact</a> ShotGrid support.\n"
+            "\n"
+            "Error: %s" % (sgtk.support_url, str(e))
```
Comment (jfboismenu) [inline python/shotgun_desktop/upgrade_startup.py:135]: Seems like we missed this one back in the day.

Comment (jfboismenu) [reply python/shotgun_desktop/upgrade_startup.py:135]: Everything below this is tk-core only and doesn't require a review.

File: python/shotgun_desktop/upgrade_startup.py [hunk 2]
```diff
@@ -127,12 +127,12 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
...
-            "There is a new update of the Shotgun Desktop, but it couldn't be installed. Shotgun "
+            "ShotGrid Desktop update failed",
+            "There is a new update of the ShotGrid Desktop, but it couldn't be installed. ShotGrid "
             "Desktop will be launched with the currently installed version of the code.\n"
-            "If this problem persists, please contact Shotgun support at "
-            "support@shotgunsoftware.com.\n"
-            "Error: %s" % str(e),
+            "If this problem persists, please <a href='%s'>contact</a> ShotGrid support.\n"
+            "\n"
+            "Error: %s" % (sgtk.support_url, str(e)),
```
Comment (jfboismenu) [inline python/shotgun_desktop/upgrade_startup.py:135]: Seems like we missed this one back in the day.

Everything below this is tk-core and does not need to be reviewed.

**Review by mathurf — APPROVED** [review]
LGTM!

Tested on Windows, mac and Linux

**Review by mathurf — CHANGES_REQUESTED** [review]
Finally I found a place where Shotgun is still used
- https://github.com/shotgunsoftware/tk-framework-desktopstartup/blob/sg-22675-shotgrid/python/shotgun_desktop/errors.py#L65-L68
- Also found that the README should be updated

File: python/shotgun_desktop/upgrade_startup.py (hunk 2 above)
Comment (mathurf) [reply python/shotgun_desktop/upgrade_startup.py:135]: also found that the README should be updated

**Comment by mathurf** [comment]
What should we do about this logo? 
Is it used? if not, should we only remove it?
https://github.com/shotgunsoftware/tk-framework-desktopstartup/blob/sg-22675-shotgrid/resources/shotgun_logo_light_medium.png

**Review by eshokrgozar — APPROVED** [review]
LGTM

**Review by mathurf — APPROVED** [review]
Thanks for the quick fixes
Everything looks good now

:shipit:

**Pull request merged**